### ISSUE TITLE
v2 describe request PackagingVersion cast exception

### ISSUE THREAD

**Issue by tamarrow** [dcos/cosmos#408]
```
Error: {u'data': None, u'message': u'com.mesosphere.universe.v3.model.V2PackagingVersion cannot be cast to com.mesosphere.universe.v3.model.V3PackagingVersion', u'type': u'unhandled_exception'}`
```

Steps to repro: 
dcos packge describe nginx 
```
Sending HTTP ['post'] to ['http://localhost:7070/package/describe']: {'Content-Type': 'application/vnd.dcos.package.describe-request+json;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.package.describe-response+json;charset=utf-8;version=v2'}
```

**Comment by cruhland** [comment]
@tamarrow are you using the latest Cosmos on `master`? I fixed a similar issue (#405) yesterday in PR #402. Here's what I get on my local machine with the latest Cosmos:

```
[charles@charles-t550 cli]$ dcos --log-level=debug package describe nginx
MainThread: 2016-06-22 11:01:48,697 /home/charles/repos/dcos/dcos-cli/dcos/http.py:_request:87 - Sending HTTP ['get'] to ['http://localhost:7070/capabilities']: {'Content-Type': 'application/vnd.dcos.capabilities+json;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.capabilities+json;charset=utf-8;version=v1'}
MainThread: 2016-06-22 11:01:48,701 /home/charles/repos/dcos/dcos-cli/cli/env/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:_new_conn:213 - Starting new HTTP connection (1): localhost
MainThread: 2016-06-22 11:01:48,710 /home/charles/repos/dcos/dcos-cli/cli/env/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:_make_request:395 - "GET /capabilities HTTP/1.1" 200 48
MainThread: 2016-06-22 11:01:48,712 /home/charles/repos/dcos/dcos-cli/dcos/http.py:_request:117 - Received HTTP response [200]: {'Content-Type': 'application/vnd.dcos.capabilities+json;charset=utf-8;version=v1', 'Content-Length': '48'}
MainThread: 2016-06-22 11:01:48,712 /home/charles/repos/dcos/dcos-cli/dcos/util.py:timer:529 - duration: dcos.http._request: 0.02s
MainThread: 2016-06-22 11:01:48,714 /home/charles/repos/dcos/dcos-cli/dcos/http.py:_request:87 - Sending HTTP ['post'] to ['http://localhost:7070/package/describe']: {'Content-Type': 'application/vnd.dcos.package.describe-request+json;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.package.describe-response+json;charset=utf-8;version=v1'}
MainThread: 2016-06-22 11:01:48,717 /home/charles/repos/dcos/dcos-cli/cli/env/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:_new_conn:213 - Starting new HTTP connection (1): localhost
MainThread: 2016-06-22 11:01:49,389 /home/charles/repos/dcos/dcos-cli/cli/env/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:_make_request:395 - "POST /package/describe HTTP/1.1" 200 3598
MainThread: 2016-06-22 11:01:49,390 /home/charles/repos/dcos/dcos-cli/dcos/http.py:_request:117 - Received HTTP response [200]: {'Content-Type': 'application/vnd.dcos.package.describe-response+json;charset=utf-8;version=v1', 'Content-Length': '3598'}
MainThread: 2016-06-22 11:01:49,390 /home/charles/repos/dcos/dcos-cli/dcos/util.py:timer:529 - duration: dcos.http._request: 0.68s
MainThread: 2016-06-22 11:01:49,392 /home/charles/repos/dcos/dcos-cli/dcos/http.py:_request:87 - Sending HTTP ['post'] to ['http://localhost:7070/package/list-versions']: {'Content-Type': 'application/vnd.dcos.package.list-versions-request+json;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.package.list-versions-response+json;charset=utf-8;version=v1'}
MainThread: 2016-06-22 11:01:49,394 /home/charles/repos/dcos/dcos-cli/cli/env/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:_new_conn:213 - Starting new HTTP connection (1): localhost
MainThread: 2016-06-22 11:01:49,412 /home/charles/repos/dcos/dcos-cli/cli/env/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:_make_request:395 - "POST /package/list-versions HTTP/1.1" 200 25
MainThread: 2016-06-22 11:01:49,414 /home/charles/repos/dcos/dcos-cli/dcos/http.py:_request:117 - Received HTTP response [200]: {'Content-Type': 'application/vnd.dcos.package.list-versions-response+json;charset=utf-8;version=v1', 'Content-Length': '25'}
MainThread: 2016-06-22 11:01:49,414 /home/charles/repos/dcos/dcos-cli/dcos/util.py:timer:529 - duration: dcos.http._request: 0.02s
{
  "description": "Nginx package",
  "framework": null,
  "licenses": [
    {
      "name": "BSD like",
      "url": "http://nginx.org/LICENSE"
    }
  ],
  "maintainer": "support@nginx.com",
  "name": "nginx",
  "packagingVersion": "2.0",
  "postInstallNotes": "Nginx has been installed.",
  "postUninstallNotes": "Nginx was uninstalled successfully.",
  "preInstallNotes": "Preparing to install nginx.",
  "scm": "http://hg.nginx.org/nginx/",
  "selected": null,
  "tags": [
    "proxy",
    "web-server",
    "cache"
  ],
  "versions": [
    "1.8.1"
  ],
  "website": null
}
[charles@charles-t550 cli]$ 
```

**Comment by tamarrow** [comment]
This is with a v2 describe request:
````
send: 'POST /package/describe HTTP/1.1\r\nHost: localhost:7070\r\nConnection: keep-alive\r\nAccept-Encoding: gzip, deflate\r\nAccept: application/vnd.dcos.package.describe-response+json;charset=utf-8;version=v2\r\nUser-Agent: python-requests/2.10.0\r\nContent-Type: application/vnd.dcos.package.describe-request+json;charset=utf-8;version=v1\r\nContent-Length: 24\r\n\r\n{"packageName": "nginx"}'
```
